### PR TITLE
Refactor: refactor code and persist user state

### PR DESCRIPTION
**Pull request by mariyasada**
refactor all code and persist user state

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,17 +1,31 @@
...
-  
+    console.log(isUserLoggedIn,"hello");
+    // if(!isUserLoggedIn)
+    // {
+    //   navigate(location?.pathname);
+    // }
+    // else{
+    //   navigate("/");
+    // }
+
```
Comment (tushartiwari7) [inline src/App.js]: remove comments

File: src/pages/login/Login.jsx [hunk 2]
```diff
@@ -23,10 +23,15 @@ export const Login = () => {
 
   const logInHandler = async (e) => {
     try {
-      e.preventDefault();
-      await dispatch(logIn(logInData)).unwrap();
-      navigate("/home");
-      toast("suceessfully login", { icon: "✔" });
+      if (logInData.email && logInData.password === "") {
```
Comment (tushartiwari7) [inline src/pages/login/Login.jsx]: In Login Page, Redirecting to homepage even if the email and password are empty.

**Review by tushartiwari7 — APPROVED** [review]
Looks clean

**Review by JantuDeb — APPROVED** [review]
Overall the code looks good

**Pull request merged**